NEWS.md

```diff
@@ -1,5 +1,9 @@
 # hpar 1.33
 
+## Changes in version 1.33.3
+
+- Update to HPA release 20.0 <2020-11-24 Thu>
```

Comment:
Add this to version 1.33.2 and remove 'Nothing yet'.

Whenever I push to Bioconductor (in this case, version 1.33.1), I bump the version on github (to show that it is potentially different), and simply say 'Nothing yet' in the NEWS file.